lib/arborist/load-virtual.js

```diff
@@ -15,6 +16,7 @@ const resolveLinks = Symbol('resolveLinks')
 const assignParentage = Symbol('assignParentage')
 const loadNode = Symbol('loadVirtualNode')
 const loadLink = Symbol('loadVirtualLink')
+const loadWorkspaces = Symbol('loadWorkspaces')
```

Comment:
I'm just noticing this now, not relevant to this PR, but we should make loadVirtual underscore its symbols like the other mixin classes.